src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindPaymentAccountsByEmailTest.java

```diff
@@ -177,7 +177,7 @@ public void returns_404_when_email_not_found() {
         Map<String, Object> response =
                 professionalReferenceDataClient.findPaymentAccountsByEmail("wrong@email.com");
 
-        assertThat(response.get("http_status")).isEqualTo("404");
+        assertThat(response.get("http_status").toString().startsWith("4"));//TODO be more specific

```

Comment:
Why removed 404 and added 4 in the above line?. Could you please check once

Reply:
Has been changed to included 404